### PR TITLE
Update README.md add shipbook as backend

### PR DESCRIPTION
**Pull request by elishas**
added shipbook as backend for the libratry

### Motivation:

added shipbook as backend for the libratry

### Modifications:

Just added it to the backend list 

### Result:

will have a fuller list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ You can choose from one of the following backends to consume your logs. If you a
 
 | Repository | Handler Description|
 | ----------- | ----------- |
+| [ShipBook/swift-log-**shipbook**](https://github.com/ShipBook/swift-log-shipbook) | a logging backend that sends log entries to [Shipbook](https://www.shipbook.io) - Shipook gives you the power to remotely gather, search and analyze your user logs and exceptions in the cloud, on a per-user & session basis. |
```
Comment (gjcairo) [inline README.md:65]: Typo:
```suggestion
| [ShipBook/swift-log-**shipbook**](https://github.com/ShipBook/swift-log-shipbook) | a logging backend that sends log entries to [Shipbook](https://www.shipbook.io) - Shipbook gives you the power to remotely gather, search and analyze your user logs and exceptions in the cloud, on a per-user & session basis. |
```

**Review by gjcairo — CHANGES_REQUESTED** [review]
Can we keep this list as append-only and put this entry at the end instead of prepending it?

**Comment by elishas** [comment]
Sure

**Comment by PeterAdams-A** [comment]
> Sure

@elishas I think Gus was expecting you to do a follow up commit to move it to the end of the list.

**Review by gjcairo — APPROVED** [review]

**Comment by PeterAdams-A** [comment]
@swift-server-bot test this please

**Pull request merged**